Week2/round-number.js

```diff
@@ -0,0 +1,18 @@
+var z = 7.25;
+console.log(z);
+var a = 7;
+console.log(a);
+console.log("the highest value is : 7.25");
+
+// second
+var a = Math.round(z);
+console.log(a);
+var c = z - a;
+if ( c > 0 && z != a ){
```

Comment:
Ah wow, cool solution!
There is a much easier way to do this, you can use [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max) it makes this a lot easier

Reply:
i will try this.

Reply:
@marcianoviereck  thanks for feedback.